src/stackit/core/wait.py

```diff
@@ -0,0 +1,61 @@
+from dataclasses import dataclass, field
+import signal
+import time
+from http import HTTPStatus
+from typing import Any, Callable, List, Tuple, Union
+
+
+@dataclass
+class WaitConfig:
+    sleep_before_wait: int = 0
+    throttle: int = 5
+    timeout: int = 30
```

Comment:
Is the throttle in seconds and the timeout in minutes? Feels a bit strange

Reply:
And maybe could we have some small description for each field?

Reply:
> Is the throttle in seconds and the timeout in minutes? Feels a bit strange

Yes. That is the same as the Go SDK

Reply:
In the Go SDK the fields are of type `time.Duration` instead of `int`. Then the user can pass them e.g. either as `5 * time.Seconds` or `1 * time.Minute`

Reply:
Ah ok, I see.
In that case this should be changes here and everything should be in seconds.